### PR TITLE
update help documentation

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

File: tools/dvpackager [hunk 1]
```diff
@@ -3,17 +3,50 @@
...
+
+ For example, the following command:
+
+ dvpackager -t -i INPUT.dv
+
+ will read INPUT.dv and rewrap those dv frames into an output file while making
+ one new output file whenever there is a change in significant technical
+ characteristics or a non-consecutive timecode value within INPUT.dv.
+
+ dvpacakager also use an 'unpackage' mode
```
Comment (kieranjol) [inline tools/dvpackager:37]: dvpackager also uses

Comment (dericed) [reply tools/dvpackager]: thx @kieranjol, updated

**Comment by dericed** [comment]
rebased after fixing an issue that @ablwr found

**Comment by ablwr** [comment]
Oh yeah as far as docs goes, this looks good to me

**Pull request merged**